inc/theme-update/astra-update-functions.php

```diff
@@ -3318,7 +3318,7 @@ function astra_set_default_breadcrumb_separator_option() {
 function astra_apply_modern_block_editor_ui() {
 	$theme_options = get_option( 'astra-settings', array() );
 
-	if ( ! isset( $theme_options['wp-blocks-ui'] ) ) {
+	if ( ! isset( $theme_options['wp-blocks-ui'] ) && ! version_compare( $theme_options['theme-auto-version'], '3.8.0', '>=' ) ) {
```

Comment:
```suggestion
	if ( ! isset( $theme_options['wp-blocks-ui'] ) && ! version_compare( $theme_options['theme-auto-version'], '3.8.0', '==' ) ) {
```
@imnavanath / @nileshc-bsf Do we also need to add condition for `'blocks-legacy-setup'`?

Reply:
No need to add the condition for `'blocks-legacy-setup'`
but `==` condition will make an issue I guess.?

@imnavanath - your thoughts on this ^

Reply:
Lets have a discussion.

Reply:
As per the discussion, modified the version check condition to `==`